code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OfferTests.java

```diff
@@ -317,19 +317,6 @@ public void testFromEventData_invalidOfferNoContent() throws Exception {
         Assert.assertNull(offer);
     }
 
-    @Test
-    public void testFromEventData_invalidOfferNoFormat() throws Exception {
```

Comment:
Why did we delete this test? It needs to be fixed by removing `Assert.assertNull(offer)` and asserting for the valid offer returned

Reply:
sure @spoorthipujariadobe 
I've added the test for the same with the name `testFromEventData_invalidOfferFormatTypeIsNotString` please have a look